packages/terra-clinical-result/src/resulttimeheadercell/ResultTimeHeaderCell.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './ResultTimeHeaderCell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed on the first line, typically abbrivated date, e.g. `Dec 12, 2010`
+   */
+  date: PropTypes.string,
+  /**
+   * Content to be displayed on the second line, typically 24 hour time in hours and minutes, e.g. `23:59`
+   */
+  time: PropTypes.string,
+  /**
+   * Visually hides the date when presented in a series of side-by-side columns of the same date.
+   */
+  hideDate: PropTypes.bool,
+  /**
+   * The padding styling to apply to the Time Column Header Cell.
+   * One of `'none'`, `'standard'`, `'compact'`.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+};
+
+const defaultProps = {
+  date: undefined,
```

Comment:
Lets remove the undefined default props

Reply:
Removed https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-db25c46a9a6a554b47090f24d2a0ebfdL29